ios-xcframework/build.sh

```diff
@@ -63,6 +66,9 @@ if [ ! -d $HERMES_XCFRAMEWORK ]; then
   exit 1
 fi
 
+# Fail early if cannot fetch version
+VERSION=$(./bin/get_gutenberg_version)
```

Comment:
I think we can use `jq` to replace this custom JSON parsing script: `jq -e -r '.version' ./package.json`?

Reply:
I considered it, but decided against it to avoid having to ensure the tool is available locally. 

Having said that... I guess it's safe to assume that to be the case and it doesn't take much to fail the script early if that's not the case 🤔

Reply:
Addressed in 8779e2e88cf17fef54b45d8a14ecbe95b1f3a167